src/main/java/com/tudu/todoapp/entities/Board.java

```diff
@@ -0,0 +1,19 @@
+package com.tudu.todoapp.entities;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
```

Comment:
no empty space. there has to be code everywhere. make it as unreadable as possible.